perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -551,11 +556,33 @@ def StartBackgroundWorkload(self):
       self.RemoteCommand(
           'nohup sysbench --num-threads=%s --test=cpu --cpu-max-prime=10000000 '
           'run 1> /dev/null 2> /dev/null &' % self.background_cpu_threads)
+    if self.background_network_mbits:
```

Comment:
Here it's particularly noticeable that the functions are getting long, and for extensibility purposes we should consider what will happen if we add more types of background workloads. It would be ideal if we could move the code for each type of background task somewhere else instead of having a list of if-checks.